extensions/community_extensions/skadnetwork.md

```diff
@@ -32,12 +44,14 @@ The responsibilities of each participant when using the SKAdNetwork specificatio
 
 1. Provide SKAdNetwork IDs to each supply partner
 2. Support OpenRTB extension objects: `BidRequest.imp.ext.skadn` & `BidResponse.imp.ext.skadn`
-3. Return all necessary signed parameters to SSP/SDK to facilitate ad signatures and receive install validation postbacks at endpoint established during SKAdNetwork registration with Apple
+3. Determine at runtime if their entity is eligible for attribution postbacks
```

Comment:
"at runtime" is a bit specific in implementation, would suggest we remove to be more flexible.

Suggested: "Determine if their entity is eligible for attribution postbacks"